src/Algorithm/pricing.jl

```diff
@@ -0,0 +1,76 @@
+####################################################################
+#                      PricingCallback
+####################################################################
+
+@with_kw struct PricingCallback <: AbstractOptimizationAlgorithm
+    stagenumber::Int = 1 
```

Comment:
```suggestion
    stage::Int = 1 
```
I suggest to change the name